ci/h2spec.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+shopt -s extglob
+
+SEP="$(tput bold)⣷$(tput sgr0)"
+LOGFILE="l5d-h2spec.log"
+SPIN='⣾⣽⣻⢿⡿⣟⣯⣷'
+
+spin() {
+    local pid=$!
+    local i=0
+    while [ "$(ps a | awk '{print $1}' | grep $pid)" ]; do
+        i=$(( (i+1) %8 ))
+        printf "\r$(tput bold)${SPIN:$i:1}$(tput sgr0)"
+        sleep .1
+    done
+}
+
+echo "${SEP} running h2spec against linkerd..."
+# if we can't find h2spec, download it
+if ! [ -e "h2spec" ] ; then
```

Comment:
consider checking for `which h2spec`, and, if not found, put it into `~/bin`, and let the user know it's there. this also avoid gitignore foo.

Reply:
yeah, was considering that - will do.